tasks/build/build.js

```diff
@@ -8,10 +8,24 @@ var watch = require('gulp-watch');
 var batch = require('gulp-batch');
 var plumber = require('gulp-plumber');
 var jetpack = require('fs-jetpack');
+var dnode = require('dnode');
 
 var bundle = require('./bundle');
 var generateSpecImportsFile = require('./generate_spec_imports');
 var utils = require('../utils');
+var ClientReceiver = require('../../app/spec_helpers/client_receiver');
+if (utils.getEnvName() === 'test') {
+    var receiver = new ClientReceiver;
+    var server = dnode({
+        specDone: receiver.specDone.bind(receiver),
+        jasmineDone: () => {
```

Comment:
We need to pass the object method by method. Dnode isn't working great with classes, but this might be something I'm just doing wrong.

https://github.com/substack/dnode/issues/180